### PR TITLE
Assignment-2b

### PR DESCRIPTION
**Pull request by inigoar**

### CODE REVIEW

File: Assignment2/inigoar-fromJSONtoJSON-LD.jsonld [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        },
+      "subjects": "http://example.org/subjects",
+      "subject": "http://example.org/subject",
+      "teacher": "http://example.org/teacher",
+      "description": {
+        "@id": "http://example.org/description",
+        "@type": "xsd:string"
+        }
+      },
+
```
Comment (paoespinozarias) [inline Assignment2/inigoar-fromJSONtoJSON-LD.jsonld:33]: You forgot to include the id.

File: Assignment2/inigoar-fromRDFtoJSON-LD.jsonld [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      "prop": "http://example.org/properties/",
+      "includes": "prop:includes",
+      "hasMeasurement": "prop:hasMeasurement",
+      "hasTemperature": "prop:hasTemperature",
+      "atTime": "prop:atTime",
+      "hasOwner": "prop:hasOwner",
+      "hasName": "prop:hasName"		
+   	},
+    
+ "Class01": {
```
Comment (paoespinozarias) [inline Assignment2/inigoar-fromRDFtoJSON-LD.jsonld:22]: You don't need to include the "Class01" attribute.

File: Assignment2/inigoar-fromRDFtoJSON-LD.jsonld [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    "@id": "http://example.org/Class01",
+    "@type": "elements:class",
+    "includes": [
+      {
+        "Sensor029": {
```
Comment (paoespinozarias) [inline Assignment2/inigoar-fromRDFtoJSON-LD.jsonld:27]: You don't need to define "Sensor029". It isn't an attribute, it is a resource, thus I suggest you to remove it. Remember that when you define a resource, you only have to use curly braces and define inside all the resource attributes. However, when you have to specify a relationship between resources is when you have to define it as an attribute, for example as you did with the attribute "hasMeasurement" for defining the relationship between Sensor029 and Measurment8401. Note that "hasMeasurement" must be defined in the @context as you have done.
This comment also applies for "Measurement8401", "Computer101", and "User10A".

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

**Comment by inigoar** [comment]
updated assignment2 and finished assignment3

**Review by inigoar — COMMENTED** [review]
updated

**Comment by paoespinozarias** [comment]
Assignment 2b is ok. Now @ocorcho will review Assignment 3

**Review by paoespinozarias — COMMENTED** [review]
ok

**Pull request merged**